libs/wingc/src/jsify.rs

```diff
@@ -873,7 +873,26 @@ impl<'a> JSifier<'a> {
 
 		let body = match &func_def.body {
 			FunctionBody::Statements(scope) => self.jsify_scope(scope, context),
-			FunctionBody::External(external_spec) => format!("return (require(require.resolve(\"{external_spec}\", {{paths: [process.env.WING_PROJECT_DIR]}}))[\"{name}\"])({parameters})")
+			FunctionBody::External(external_spec) => {
```

Comment:
@MarkMcCulloh I am wondering if this mean that we can avoid the copying of extern files altogether, no? (if I understand correctly we are still doing them).

Reply:
My last extern PR already avoided copying or creating any sort of wrapper file.